app/generators/questionnaire_generator.rb

```diff
@@ -27,12 +27,16 @@ def substitute_variables(response_id, title, content)
       response = Response.find_by_id(response_id) # allow nil response id for preview
       return [title, content] if response.blank?
       student, mentor = response.determine_student_mentor
+      subs_hash = {
+        mentor_title: mentor&.role&.title,
+        mentor_gender: mentor&.gender,
+        mentor_name: mentor&.first_name,
+        organization: student.role.organization.name,
```

Comment:
ik zag in de import ook mentoren zonder studenten staan. Zouden we hier dan ook een & moeten gebruiken?

Reply:
Nee.